.github/workflows/installation-tests.yml

```diff
@@ -56,14 +56,47 @@ jobs:
           name: ${{ matrix.image }}-image
           path: ${{ matrix.image }}.tar
           retention-days: 1
+  
+  build-mock-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [vllm-mock]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build vllm-mock
+        run: |
+          cd development/app
+          docker build -t aibrix/vllm-mock:${{ github.sha }} -f Dockerfile .
+
+      - name: Save image
+        run: |
+          docker save aibrix/vllm-mock:${{ github.sha }} > vllm-mock.tar
+
+      - name: Upload image artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: vllm-mock-image
+          path: vllm-mock.tar
```

Comment:
Did you check this the file size? It may fill up the artifact storage pretty quick. the cleaner version is pretty small, but I remember some simulator libs are added and then it becomes large. I kind of forget whether we use same dependency file or not. could you help double check it?

Reply:
good catch. I checked, size is 1.6G and retention-days that can be set is 1 day. 

Checking, if there is a post job cleanup function to delete the artifact.

Reply:
1.6B was on local mac. For github action it is 400MB.

<img width="1575" alt="image" src="https://github.com/user-attachments/assets/e40be179-739c-43ad-b6f2-18498fd96336" />